src/NServiceBus.Core/Hosting/Helpers/AssemblyScanningComponent.cs

```diff
@@ -21,6 +23,12 @@ public static AssemblyScanningComponent Initialize(Configuration configuration,
                 var directoryToScan = AppDomain.CurrentDomain.RelativeSearchPath ?? AppDomain.CurrentDomain.BaseDirectory;
 
                 assemblyScanner = new AssemblyScanner(directoryToScan);
+
+                if (!string.IsNullOrWhiteSpace(assemblyScannerSettings.AdditionalAssemblyScanningPath))
```

Comment:
Since you are validating the value in the scanner, I don't think this needs to be done here, so this code can be simplified, and the setting can move down to where the others are.